experiments/react-native-uifabric/package.json

```diff
@@ -30,21 +30,32 @@
     "@uifabricshared/immutable-merge": "0.2.2",
     "@uifabricshared/theming-ramp": "0.5.2",
     "@uifabricshared/theming-react-native": "0.3.2",
-    "tslib": "^1.9.3"
+    "tslib": "^1.10.0"
   },
   "devDependencies": {
     "@office-iss/react-native-win32": "0.27.1",
     "@office-iss/rex-win32": "0.0.34",
-    "@types/prop-types": "15.5.1",
+    "@types/prop-types": "^15.5.10",
     "@types/react": "16.9.11",
     "@types/react-dom": "16.9.5",
     "@types/react-native": "0.60.19",
     "@uifabricshared/build-native": "^0.1.1",
     "@uifabricshared/eslint-config-rules": "^0.1.1",
-    "office-ui-fabric-react": "^7.28.2",
     "metro-react-native-babel-preset": "0.58.0",
     "react": "16.8.6",
-    "react-native": "^0.60.0"
+    "react-native": "^0.60.0",
+    "es6-map": "^0.1.5",
+    "es6-weak-map": "^2.0.2",
+    "@uifabric/foundation": "^7.5.3",
+    "@uifabric/icons": "^7.3.2",
+    "@uifabric/merge-styles": "^7.8.3",
+    "@uifabric/react-hooks": "^7.0.3",
+    "@uifabric/set-version": "^7.0.3",
+    "@uifabric/styling": "^7.10.2",
```

Comment:
what are these dependencies? Do we need them?

Reply:
removing w/ abandoned oufr dependency removal (jasonmo has done it separately)